engines/grim/font.cpp

```diff
@@ -175,7 +177,55 @@ void BitmapFont::load(const Common::String &filename, Common::SeekableReadStream
 	g_driver->createFont(this);
 }
 
-uint16 BitmapFont::getCharIndex(uint16 c) const {
+void BitmapFont::loadTGA(const Common::String &filename, Common::SeekableReadStream *index, Common::SeekableReadStream *image) {
+  	Image::TGADecoder dec;
+	bool success = dec.loadStream(*image);
+
+	if (!success)
+		return;
+
+	const Graphics::Surface *surf = dec.getSurface();
+
+	_filename = filename;
+	_numChars = index->size() / 10;
+	_dataSize = surf->w * surf->h;
+	_kernedHeight = 16;
+	_baseOffsetY = 0;
+	_firstChar = 0;
+	_lastChar = 0xffff;
+
+	_isDBCS = false;
+	_isUnicode = true;
+
+	// Read character headers
+	_charHeaders = new CharHeader[_numChars];
+
+	// Read character indexes - are the key/value reversed?
+	_fwdCharIndex.resize(0x10000, -1);
```

Comment:
what 0x10000 magic number do? add macro?

Reply:
It's highest uint16 + 1. Added a macro